community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/centrality/EigenvectorCentralityPower.java

```diff
@@ -90,6 +91,14 @@ public EigenvectorCentralityPower( Direction relationDirection,
      */
     public Double getCentrality( Node node )
     {
+        if ( !nodeSet.contains( node ) )
```

Comment:
Almost all changes in `EigenvectorCentralityPower` are exactly the same as for `EigenvectorCentralityArnoldi`. Looks like we can extract superclass for those 2?